Assets/Scripts/BoundaryController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(Collider2D))]
+public class BoundaryController : MonoBehaviour
+{
+
+    private Collider2D _collider;
+    public const int SCREEN_WIDTH = 32;
+    public const int SCREEN_HEIGHT = 18;
+    public const int SCREEN_MIN_X = -(SCREEN_WIDTH / 2);
+    public const int SCREEN_MAX_X = SCREEN_WIDTH / 2;
+    public const int SCREEN_MIN_Y = -(SCREEN_HEIGHT / 2);
+    public const int SCREEN_MAX_Y = SCREEN_HEIGHT / 2;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        _collider = GetComponent<Collider2D>();
+    }
+
+    void LateUpdate()
```

Comment:
A comment explaining what is going on

Reply:
Good suggestion, I'll add a comment here.

Reply:
Use triple slash doc strings for comments like this. It should generate an `<summary>` enclosure for your doc. This allows the IDE to index the documentation.

Reply:
Oh, I didn't know about this. I tried it out, and it seems handy! Thanks for the new knowledge!